master/getting-started/kubernetes/tutorials/simple-policy.md

```diff
@@ -19,30 +19,38 @@ kubectl create ns policy-demo
 
 We'll use Kubernetes `Deployment` objects to easily create pods in the `Namespace`.
 
-1) Create some nginx pods in the `policy-demo` Namespace, and expose them through a Service.
+1. Create some nginx pods in the `policy-demo` Namespace.
 
-```shell
-# Run the Pods.
-kubectl run --namespace=policy-demo nginx --replicas=2 --image=nginx
+   ```shell
+   kubectl run --namespace=policy-demo nginx --replicas=2 --image=nginx
+   ```
 
-# Create the Service.
-kubectl expose --namespace=policy-demo deployment nginx --port=80
-```
+1. Expose them through a service.
 
-2) Ensure the nginx service is accessible.
+   ```bash
+   kubectl expose --namespace=policy-demo deployment nginx --port=80
+   ```
 
-```
-# Run a Pod and try to access the `nginx` Service.
-$ kubectl run --namespace=policy-demo access --rm -ti --image busybox /bin/sh
-Waiting for pod policy-demo/access-472357175-y0m47 to be running, status is Pending, pod ready: false
+1. Ensure the nginx service is accessible.
 
-If you don't see a command prompt, try pressing enter.
+   ```bash
+   kubectl run --namespace=policy-demo access --rm -ti --image busybox /bin/sh
+   ```
 
-/ # wget -q nginx -O -
-```
+   It should return something like the following.
```

Comment:
rather than return, this should say "You will receive a command line prompt within the pod as shown below" or something like that.